client/src/store/api.ts

```diff
@@ -1,5 +1,19 @@
 import axios from 'axios';
+import {TOKEN} from './login/setToken.helper';
 
-export default axios.create({
+export const API = axios.create({
+    baseURL: process.env.REACT_APP_BASE_API,
+    headers: {
+        'Content-Type': 'application/json',
+    },
+});
+
+API.interceptors.request.use((config: any) => {
+    const token = localStorage.getItem(TOKEN);
+    config.headers['x-access-token'] =  token || '';
+    return config;
+});
+
+export const AuthAPI = axios.create({
```

Comment:
maybe API here and AuthAPI in the bottom?

Reply:
Done